### PR TITLE
[lapresse] new extractor

### PR DESCRIPTION
**Pull request by up4**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [ ] Improvement
- [x] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Here is a new extractor for what was originally the biggest french language print media in America, [La Presse](https://www.lapresse.ca) from Montreal, Quebec, Canada. Video detail pages only for now.

### CODE REVIEW

File: youtube_dl/extractor/lapresse.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        'url': 'http://www.lapresse.ca/videos/actualites/201610/26/46-1-louisville-ou-la-vie-en-noir-et-blanc.php/c28cee30286f4c53ba4f62459dca4a7b',
+        'info_dict': {
+            'id': 'c28cee30286f4c53ba4f62459dca4a7b',
+            'ext': 'mp4',
+            'title': 'Louisville ou la vie en noir et blanc'
+        }
+    }
+
+    def _real_extract(self, url):
+        id = self._match_id(url)
```
Comment (dstftw) [inline youtube_dl/extractor/lapresse.py]: Do not shadow built-in names.

**Comment by up4** [comment]
all requested fixes are in

**Comment by dstftw** [comment]
I've already pointed out numerous of times: check code with flake8.

**Comment by up4** [comment]
I don't give a flying f*ck if you don't want my code. You're not doing me a favour. I am. And it's the last time.

**Comment by dstftw** [comment]
I don't either.

**Comment by up4** [comment]
I'm not worried. You will, [eventually](https://twitter.com/MUP4/status/800051956446330880).

**Comment by dstftw** [comment]
When at Rome, do as the Romans do.

**Pull request closed without merging**